src/routes/adoption/+server.ts

```diff
@@ -0,0 +1,4 @@
+export function GET() {
```

Comment:
this is carried over from the other pr, isn't it?

Reply:
Good catch! Removed